pkgs/shells/fish/babelfish.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, stdenv, buildGoModule, fetchFromGitHub }:
+buildGoModule rec {
+  pname = "babelfish";
+  version = "0.1.3";
+
+  src = fetchFromGitHub {
+    owner = "bouk";
+    repo = "babelfish";
+    rev = "v${version}";
+    sha256 = "08i4y4fw60ynamr1jz8nkfkidxj06vcyhi1v4wxpl2macn6n4skk";
+  };
+
+  vendorSha256 = "0xjy50wciw329kq1nkd7hhaipcp4fy28hhk6cdq21qwid6g21gag";
+
+  meta = with lib; {
+    description = "Translate bash scripts to fish";
+    homepage = "https://github.com/bouk/babelfish";
+    license = licenses.mit;
+    maintainers = with maintainers; [ bouk kevingriffin ];
```

Comment:
FYI, if `meta.platforms` is not set, the package won't be substitutable from the cache. Fixed in https://github.com/NixOS/nixpkgs/pull/109178.

Reply:
Platforms is set in `buildGoModule` so it will be built and cached by hydra.

https://github.com/NixOS/nixpkgs/blob/213286bb181c9b6588bb5c4fa130301cefc90dd0/pkgs/development/go-modules/generic/default.nix#L236

https://hydra.nixos.org/jobset/nixpkgs/trunk/jobs-tab?filter=babelfish

Reply:
Ah, I see.

Well, still better to be explicit about it (especially considering the manual says "Platforms should be set").